Manual.md

```diff
@@ -0,0 +1,254 @@
+
+```
+export PGARCHIVE=/path/to/container
+```
+
+For high-load DBs at emarsys.com the following btrfs mount options work best. You do need `user_subvol_rm_allowed` for snapshot expiry to work, the rest merely improve performance for us. See [btrfs mount options] for more details.
+
+```
+user_subvol_rm_allowed,noatime,nobarrier,enospc_debug,nodatacow,metadata_ratio=6
+```
+
+Now you need to use `pgarchive container init`, possibly with more configuration, to create container(s). Re-starting containers on server re-boots is left up to you. (TODO: add sysV and systemd service scripts to repository)
+
+### Security
+
+There are no internal security mechanisms at all. Note especially:
+
+* This is expected to be run on a trusted network. You could secure the upstream DB connections using SSL, but this is up to you.
+* By default backup containers can log into their upstream DB server using a privileged user, although you could reduce this to replication only after container initialization.
+* Access to containers and their data is purely host based. Data is not encrypted.
+
+### Monitoring Hints
+
+Backup services need monitoring. How exactly this is done is out of scope of pgarchive and determined by local policy, but here is what we found useful at emarsys.com, for each container:
+
+* The wal_archive directory should have frequent activity (mtime). If your upstream DB has long idle periods setting [archive_timeout] will put a limit on this (like 1 hour). Typically the wal_archive subsystem is not running if this alert is raised, or otherwise cannot write to the wal_archive directory.
+
+* The snapshots directory should have frequent activity too, depending on the cron job for snapshot creation. Note snapshot creation only succeeds if the standby process is active, which means running and having a recent restartpoint, so this typically flags standby process problems.
+
+* You absolutely need to monitor the size of your upstream DB's $PGDATA/pg_xlog directory. But you already do this, don't you? If pgarchive containers cannot stream replication data, old WAL segments cannot be released there because it's held by the container's [replication slot][replication slots]. You may eventually need to decide to drop the replication slot, which will break synchronization with your container.
+
+Of course you should monitor free disk space for your containers, and other common system parameters.It's probably less useful to check for the presence of [pg_receivexlog] and PostgreSQL standby processes, because it's hard to associate them with multiple containers easily, and this may miss certain problems if subsystems become stuck without dying fully.
+
+
+## Architecture Overview
+
+The basic components and data flow for a single container looks like this. Some details like automatic expiry and btrfs filesystem maintenance are not shown for clarity.
+
```

Comment:
This picture we should put at the beginning IMHO. It would make easier to understand the previous sections.